core/Intl/Data/Resources/currencies.php

```diff
@@ -179,5 +176,4 @@
     'XOF' => array('Fr', 'West African CFA franc'),
     'YER' => array('﷼', 'Yemeni rial'),
     'ZMW' => array('ZK', 'Zambian kwacha'),
-    'ZWL' => array('$', 'Zimbabwean dollar'),
```

Comment:
Did you check if removing/changing such currencies might break Matomo if a site had one of those chosen beofre? If so, we maybe should provide an update script to update all sites using one of the changed/removed currencies...